### PR TITLE
+crossorigin="anonymous" && +rel="noreferrer"

### PR DESCRIPTION
**Pull request by jodumont**
update source after @DanielRuf recommendation
https://github.com/insites/cookieconsent/pull/368#issuecomment-394489675

### CODE REVIEW

File: src/cookieconsent.js [hunk 1]
```diff
@@ -34,7 +34,7 @@
       var value = '; ' + document.cookie;
       var parts = value.split('; ' + name + '=');
       return parts.length != 2 ?
-        undefined : parts.pop().split(';').shift();
+        undefined : parts.pop().split(';').shift();crossorigin="anonymous" rel="noreferrer"
```
Comment (DanielRuf) [inline src/cookieconsent.js]: I don't think this works like this. Also please use a new line. Should this be a comment?

File: src/cookieconsent.js [hunk 2]
```diff
@@ -222,11 +222,11 @@
...
+        messagelink: '<span id="cookieconsent:desc" class="cc-message">{{message}} <a aria-label="learn more about cookies" role=button tabindex="0" class="cc-link" href="{{href}}" rel="noopener noreferrer nofollow" target="_blank" crossorigin="anonymous">{{link}}</a></span>',
         dismiss: '<a aria-label="dismiss cookie message" role=button tabindex="0" class="cc-btn cc-dismiss">{{dismiss}}</a>',
         allow: '<a aria-label="allow cookies" role=button tabindex="0"  class="cc-btn cc-allow">{{allow}}</a>',
         deny: '<a aria-label="deny cookies" role=button tabindex="0" class="cc-btn cc-deny">{{deny}}</a>',
-        link: '<a aria-label="learn more about cookies" role=button tabindex="0" class="cc-link" href="{{href}}" target="_blank">{{link}}</a>',
+        link: '<a aria-label="learn more about cookies" role=button tabindex="0" class="cc-link" href="{{href}}" rel="noopener noreferrer nofollow" target="_blank" crossorigin="anonymous">{{link}}</a>',
```
Comment (DanielRuf) [inline src/cookieconsent.js]: nofollow and crossorigin are no security issue per se but noopener.

Comment (DanielRuf) [reply src/cookieconsent.js]: Please just add `noopener`.

**Comment by jodumont** [comment]
the rel="noopener noreferrer nofollow" was already there so
thank for you're help but it's become a pain in the ass to try to understand what you really want

**Pull request closed without merging**